src/components/textarea/textarea-test.stories.tsx

```diff
@@ -92,6 +92,11 @@ export default {
         type: "text",
       },
     },
+    borderRadius: {
```

Comment:
**praise**: nice work updating the `controls` for the test stories - it's something I always forget, but I've confirmed this works nicely 👍

Reply:
Cheers. I asked @sianford if having it there would be of benefit to QA (since the issues with CSB cropped up) and she agreed it would help the QA lot out.